src/main/kotlin/net/atos/zac/zoeken/IndexingService.kt

```diff
@@ -300,4 +296,14 @@ class IndexingService @Inject constructor(
             throw IndexingException(ioException)
         }
     }
+
+    private fun <T> runIgnoringExceptions(objectType: ZoekObjectType, fn: () -> T): T? {
```

Comment:
We are not really ignoring them. Far from that I would say. But we continue anyway. ;-) Not sure how best to phrase that..

Reply:
Maybe `continuingOnExceptions`.. not sure

Reply:
Renamed to `continueOnExceptions`